python/spacewalk/rhn-conf/rhn_server_satellite.conf

```diff
@@ -43,3 +43,6 @@ default_mail_from =
 reposync_download_threads = 5
 reposync_timeout = 300
 reposync_minrate = 1000
+
+# URLGrabber log level
```

Comment:
Could you please extend comment with maybe a couple of examples what could be specified here.